File: src/test/java/ru/ezhov/rocket/action/notification/PopupNotificationServiceTest.java
package ru.ezhov.rocket.action.notification;

public class PopupNotificationServiceTest {
    public static void main(String[] args) {
        PopupNotificationService popupNotificationService = new PopupNotificationService();
        for (int i = 0; i < 5; i++) {

            try {
                Thread.sleep(500);
            } catch (InterruptedException e) {
                e.printStackTrace();
            }
            popupNotificationService.show("" + i);
        }
    }
}

File: src/test/kotlin/ru/ezhov/rocket/action/notification/PopupNotificationServiceTest.kt
package ru.ezhov.rocket.action.notification

object PopupNotificationServiceTest {
    @JvmStatic
    fun main(args: Array<String>) {
        val popupNotificationService = PopupNotificationService()
        for (i in 0..4) {
            try {
                Thread.sleep(500)
            } catch (e: InterruptedException) {
                e.printStackTrace()
            }
            popupNotificationService.show(NotificationType.INFO, "" + i)
            popupNotificationService.show(NotificationType.WARN, "" + i)
            popupNotificationService.show(NotificationType.ERROR, "" + i)
        }
    }
}